packages/cli/package.json

```diff
@@ -40,15 +41,18 @@
     "src"
   ],
   "dependencies": {
+    "chalk": "^5.4.1",
     "commander": "^10.0.0",
     "comment-json": "^4.2.3",
     "inquirer": "^8.2.5",
+    "node-fetch": "^3.3.2",
     "package-manager-detector": "^0.2.0"
```

Comment:
Nit: This library is not being used anywhere in the current code.

It was added [here](https://github.com/wrtnlabs/agentica/commit/40c205ad14e2106cec1e22b9284a1e94a0535b5f#diff-2fb448d11702324cc59d657d5945d59cd325540eb87b71f2e66a5bc01ada834c), but it is not used here either.
I just discovered it now and wanted to bring it up.

Reply:
And, If you want to use `fetch` function, Just use it. `fetch` is core function of node.js now.